### PR TITLE
Change two cases to plural: basinSubzones

### PR DESCRIPTION
**Pull request by alegnaaived**

### CODE REVIEW

**Comment by cazapatamar** [comment]
Sorry, it was my mistake by not testing completely when i worked on this endpoints. Two issues i see with this PR:

1. It should be merged into release branch instead of develop. Merge or rebase, push and change base branch in this PR.
2. Add the solution in function getSCIHFPALayer also because it has the same mistake.

File: services/forest/src/service/SCIHF.js [hunk 1]
```diff
@@ -81,7 +81,7 @@ module.exports = (SCIHFPersistence, restAPI) => {
         case 'states':
           data = await SCIHFPersistence.findSCIHFLayerInState(areaId);
           break;
-        case 'basinSubzone':
+        case 'basinSubzones':
```
Comment (cazapatamar) [inline services/forest/src/service/SCIHF.js:84]: Fix it also in getSCIHFPALayer please

**Review by cazapatamar — CHANGES_REQUESTED** [review]

File: services/forest/src/service/SCIHF.js (hunk 1 above)
Comment (alegnaaived) [reply services/forest/src/service/SCIHF.js:84]: Done

**Review by erikasv — CHANGES_REQUESTED** [review]
This PR is including changes not ready for release, I believe you should close this PR, create a new branch from release and reproduce the changes there, so it includes the fix but not the new service.

**Comment by erikasv** [comment]
Also another option is to rebase to release if you haven't done any merge

**Comment by alegnaaived** [comment]
To remove this branch

**Pull request closed without merging**